usr/lib/pkcs11/cca_stdll/cca_specific.c

```diff
@@ -2858,6 +2859,14 @@ static CK_RV rsa_import_privkey_crt(TEMPLATE * priv_tmpl)
         return rc;
     }
 
+    OPENSSL_cleanse(p_prime->pValue, p_prime->ulValueLen);
+    OPENSSL_cleanse(q_prime->pValue, q_prime->ulValueLen);
+    OPENSSL_cleanse(dmp1->pValue, dmp1->ulValueLen);
+    OPENSSL_cleanse(dmq1->pValue, dmq1->ulValueLen);
+    OPENSSL_cleanse(iqmp->pValue, iqmp->ulValueLen);
+    OPENSSL_cleanse(pub_exp->pValue, pub_exp->ulValueLen);
```

Comment:
Dont cleanse non-sensitive data, see my comment.

Reply:
Fixed.

As far as I can tell, the 2 primes (p, q) as well as the 2 exponents and the coefficient are sensitive for RSA CRT. The public exponent and the modulus are not sensitive. Please correct me if I am wrong.